cocos2d/core/platform/CCScreen.js

```diff
@@ -101,7 +101,7 @@ cc.screen = /** @lends cc.screen# */{
      * @returns {Boolean}
      */
     fullScreen: function() {
-	    return this._supportsFullScreen && document[ this._fn.fullscreenEnabled ];
+	    return this._supportsFullScreen && document[this._fn.fullscreenElement];
```

Comment:
@pandamicro, could you please clarify?
As I see, the `_fnMap` already completed to fit all browsers.
But, regarding to `_fnMap`, I think that structure of `_fnMap` attribute should be changed and initialization of `_fn` attribute refactored (function `this.init()`).